From e31b4f15922947241d59972876310d12506e4677 Mon Sep 17 00:00:00 2001
From: maguoqun <guoqun.ma@spacemit.com>
Date: Thu, 23 May 2024 18:16:53 +0800
Subject: [PATCH 0909/1205] MUSE-N1: u3 set the default mode to host 1.so 2.5G
 eth enable default 2.bianbu-uas deb do check udisk quick mode in initramfs

Change-Id: Ib7ff9e8e04105f3755d810e4e471dfd77df1ed6b
---
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts
index 0742e998c946..5ee252248309 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts
@@ -678,7 +678,7 @@ dwc3@c0a00000 {
 		snps,dis-del-phy-power-chg-quirk;
 		snps,dis-tx-ipgap-linecheck-quirk;
 		usb-role-switch;
-		role-switch-default-mode = "peripheral";
+		role-switch-default-mode = "host";
 	};
 };
 
-- 
2.47.0

